ts/screens/LoadingScreen.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { View } from "react-native";
+import LoadingSpinnerOverlay from "../components/LoadingSpinnerOverlay";
+import BaseScreenComponent from "../components/screens/BaseScreenComponent";
+
+interface State {
+  isLoading: boolean;
+}
+/**
+ * A loading screen used during the saga startup process to display a Spinner
+ * until all the saga are performed, including the navigation in the message screen
+ */
+class LoadingScreen extends React.PureComponent<never, State> {
```

Comment:
```suggestion
class OnboardingCompleteLoadingScreen extends React.PureComponent<never, State> {
```
I think it is more appropriated